activesupport/lib/active_support/core_ext/date/conversions.rb

```diff
@@ -22,29 +22,29 @@ class Date
 
   # Convert to a formatted string. See DATE_FORMATS for predefined formats.
   #
-  # This method is aliased to <tt>to_fs</tt>.
+  # This method is aliased to <tt>to_formatted_s</tt>.
   #
   #   date = Date.new(2007, 11, 10)       # => Sat, 10 Nov 2007
   #
-  #   date.to_formatted_s(:db)            # => "2007-11-10"
+  #   date.to_fs(:db)            # => "2007-11-10"
   #   date.to_fs(:db)                     # => "2007-11-10"
```

Comment:
```suggestion
  #   date.to_formatted_s(:db)                     # => "2007-11-10"
```

since this part of the documentation is to show the alias